examples/evm/nft-linker/NftLinker.sol

```diff
@@ -19,7 +19,11 @@ contract NftLinker is ERC721, AxelarExecutable, Upgradable {
     string public chainName; //To check if we are the source chain.
     IAxelarGasService public immutable gasService;
 
-    constructor(address gateway_, address gasReceiver_) ERC721('Axelar NFT Linker', 'ANL') AxelarExecutable(gateway_) {
+    constructor(
+        address gateway_,
+        address gasReceiver_,
+        address owner_
+    ) ERC721('Axelar NFT Linker', 'ANL') AxelarExecutable(gateway_) Upgradable(owner_) {
```

Comment:
```suggestion
    ) ERC721('Axelar NFT Linker', 'ANL') AxelarExecutable(gateway_) Upgradable(msg.sender) {
```
Just use msg.sender for these example contracts for simplicity and avoids breaking compatibility